swagger/extensible-enums-swagger.json

```diff
@@ -0,0 +1,118 @@
+{
+ "swagger": "2.0",
+ "info": {
+  "version": "2016-07-07",
+  "title": "PetStore Inc",
+  "description": "PetStore"
+ },
+ "host": "localhost",
+ "schemes": [
+  "https"
+ ],
+ "paths": {
+  "/pet/extensibleenums/{petId}": {
+   "get": {
+    "operationId": "Pet_GetByPetId",
+    "parameters": [
+     {
+      "$ref": "#/parameters/petid"
+     }
+    ],
+    "responses": {
+     "200": {
+      "schema": {
+       "$ref": "#/definitions/Pet"
+      }
+     }
+    },
+    "produces": [
+     "application/json"
+    ],
+    "consumes": [
+     "application/json"
+    ]
+   }
+  }
+ },
+ "definitions": {
+  "Pet": {
+   "properties": {
+    "name": {
+     "type": "string"
+    },
+    "DaysOfWeek": {
+     "type": "string",
+     "description": "Type of Pet",
+     "enum": [
+      "Monday",
+      "Tuesday",
+      "Wednesday",
+      "Thursday",
+      "Friday",
+      "Saturday",
+      "Sunday"
+     ],
+     "x-ms-enum": {
+      "name": "DaysOfWeekExtensibleEnum",
+      "modelAsString": true
+     },
+     "default": "Friday"
+    },
+    "IntEnum": {
```

Comment:
how would this compare to "simply" modelling it as
``` JSON
"enum": [
 "1",
 "1.1",
 "1.2",
 "1.3",
 "2",
 "2.1",
 "2.2",
 "3",
 "3.1",
 "3.3"
],
"x-ms-enum": {
 "modelAsString": true,
 "name": "IntEnum",
 "values": [
  {
   "value": "1",
   "description": "one",
   "name": "1"
  },
  {
   "value": "1.1",
   "description": "one",
   "name": "1"
  },
  {
   "value": "1.2",
   "description": "one",
   "name": "1"
  },
  {
   "value": "1.3",
   "description": "one",
   "name": "1"
  },
  {
   "value": "2",
   "description": "two",
   "name": "2"
  },
  {
   "value": "2.1",
   "description": "two",
   "name": "2"
  },
  {
   "value": "2.2",
   "description": "two",
   "name": "2"
  },
  {
   "value": "3",
   "description": "three",
   "name": "3"
  },
  {
   "value": "3.1",
   "description": "three",
   "name": "3"
  },
  {
   "value": "3.3",
   "description": "three",
   "name": "3"
  }
 ]
}
```
It's slightly more verbose but:
- uses the syntax we already have (which supports mapping multiple values to the same name!)
- the bare Swagger doesn't lie (the alternative values may appear on the wire, so they should appear in the `enum`)